distributionviewer/core/static/js/app/components/containers/chart-line-container.js

```diff
@@ -9,13 +9,13 @@ export default class extends React.Component {
   componentDidMount() {
     let props = this.props;
     let line = d3Shape.line()
-                .x(d => {return props.xScale(d.x);})
-                .y(d => {return props.yScale(d.y);});
+                .x(d => props.xScale(d.x))
```

Comment:
Heh thanks for this. I had a fix for it locally but ended up stashing it due to other detail page related work that broke things.